.gitignore

```diff
@@ -74,3 +74,4 @@ webapp/WEB-INF/web.xml
 webapp/xqts/config.xml
 tools/yajsw/logs/wrapper.log
 tools/yajsw/logs/wrapper.log.lck
+Project_Default.xml
```

Comment:
it was added by intellij itself ; I guess this file should never be shared with users (it is in the root directory, which is highly not intellij-ish